modules/timeplus/src/main/java/org/testcontainers/timeplus/TimeplusContainer.java

```diff
@@ -0,0 +1,120 @@
+package org.testcontainers.timeplus;
+
+import org.testcontainers.containers.JdbcDatabaseContainer;
+import org.testcontainers.containers.wait.strategy.Wait;
+import org.testcontainers.utility.DockerImageName;
+
+import java.time.Duration;
+import java.util.HashSet;
+import java.util.Set;
+
+/**
+ * Testcontainers implementation for Timeplus.
+ */
+public class TimeplusContainer extends JdbcDatabaseContainer<TimeplusContainer> {
+
+    private static final String NAME = "timeplus";
+
+    private static final DockerImageName TIMEPLUS_IMAGE_NAME = DockerImageName.parse("timeplus/timeplusd:2.3.3");
```

Comment:
please use `timeplus/timeplusd:develop`

Reply:
no we shouldn't use latest image, use stable version instead

Reply:
https://github.com/testcontainers/testcontainers-java/blame/18a7c277bc749057bf5783a50ee3d1a052fab022/docs/contributing.md#L54

Reply:
Let's removed. New modules do not provide an specific tag. It should be provided by the consumer.

```suggestion
```